python/Ganga/test/GPI/LCG/TestCREAM.py

```diff
@@ -25,7 +25,7 @@ def test_job_kill(gpi):
     #    1176      30       1007          587     420 ce01.tier2.hep.manchester.ac.uk:8443/cream-pbs-long
     #
     # Select the CREAM CEs (URL path starts with '/cream') and how many free slots they have
-    ces = re.findall(r'^\s*\d+\s*(?P<free>\d+)\s*\d+\s*\d+\s*\d+\s*(?P<ce>[^:/\s]+:\d+/cream.*)$', stdout, re.MULTILINE)
+    ces = re.findall(r'^\s*\d+\s*(?P<free>\d+)\s*\d+\s*\d+\s*\d+\s*(?P<ce>[^:/\s]+uk:\d+/cream.*)$', stdout, re.MULTILINE)
```

Comment:
Had an issue of some odd CE hanging on submission. I'm going to assume the UK ones are more reliable :smile: